handlers/handlers.go

```diff
@@ -34,17 +36,57 @@ func GeographyStart(cfg config.Config, rc RenderClient) http.HandlerFunc {
 // GetArea Handler
 func GetArea(ctx context.Context, cfg config.Config, c Clients) http.HandlerFunc {
 	return dphandlers.ControllerHandler(func(w http.ResponseWriter, req *http.Request, lang, collectionID, accessToken string) {
-		func(w http.ResponseWriter, req *http.Request, lang, collectionID, accessToken string) {
-			vars := mux.Vars(req)
-			areaID := vars["id"]
-			//acceptedLang := req.Header.Get("Accepted-Language")
-			areaData, err := c.AreaApi.GetArea(ctx, accessToken, "", collectionID, areaID)
-			if err != nil {
-				log.Error(ctx, "Fetching Area Data", err)
-			}
-			basePage := c.Render.NewBasePageModel()
-			model := mapper.CreateAreaPage(basePage, areaData)
-			c.Render.BuildPage(w, model, "area-summary")
-		}(w, req, lang, collectionID, accessToken)
+		GetAreaViewHandler(w, req, ctx, c, lang, collectionID, accessToken)
 	})
 }
+
+func GetAreaViewHandler(w http.ResponseWriter, req *http.Request, ctx context.Context, c Clients, lang, collectionID, accessToken string) {
+	var err error
+	var relationsErr error
+	var ancestorErr error
+	var areaData areas.AreaDetails
+	var relationsData []areas.Relation
+	var ancestorData []areas.Ancestor
```

Comment:
consider writing it like this: 
```
  var (    
        err error
	relationsErr error
	ancestorErr error
        areaData areas.AreaDetails
	relationsData []areas.Relation
	ancestorData []areas.Ancestor
   )
```

Reply:
Thanks Peter, I will do that on the next PR pass.